storage/statedb/cache_hybrid_test.go

```diff
@@ -3,14 +3,15 @@ package statedb
 import (
 	"bytes"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func getTestHybridConfig() TrieNodeCacheConfig {
 	return TrieNodeCacheConfig{
 		CacheType:          CacheTypeHybrid,
-		LocalCacheSizeMB:   1024 * 1024,
+		LocalCacheSizeMB:   1024,
```

Comment:
Is this 1GB?? Is it too much for testing?

Reply:
Changed it to 100 MB